src/style.css

```
@@ -0,0 +1,84 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+}
+
+.flex-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.flex-item {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  flex-basis: 300px;
+  flex-grow: 1;
+  font-size: 100px;
+  color: #fff;
+  text-align: center;
+}
+
+.flex-item-1 {
+  background-color: #f00;
+  order: 1;
+}
+
+.flex-item-2 {
+  background-color: #c00;
+  order: 2;
+}
+
+.flex-item-3 {
+  background-color: #900;
+  order: 3;
+}
+
+.flex-item-4 {
+  background-color: #600;
+  order: 4;
+}
+
+.flex-item-5 {
+  background-color: #300;
+  order: 5;
+}
+
+.flex-item-6 {
+  background-color: #000;
+  order: 6;
+}
+
+@media (min-width: 600px) {
+  .flex-container {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .flex-item-4 {
+    order: 3;
+  }
+
+  .flex-item-3 {
+    order: 4;
+  }
+}
+
+@media (min-width: 900px) {
+  .flex-item {
+    min-width: calc(100vw / 3);
```

```suggestion
    flex-basis: 33.33%
```
let's use flex-basis instead of min width